### PR TITLE
Bug 1317989 - Move bidi wrapping back into MessageContext

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1317989

### CODE REVIEW

File: src/intl/resolver.js [hunk 1]
```diff
@@ -329,15 +334,20 @@ function Pattern(env, ptn) {
...
             'Too many characters in placeable ' +
             `(${str.length}, max allowed is ${MAX_PLACEABLE_LENGTH})`
           )
         );
-        result += str.substr(0, MAX_PLACEABLE_LENGTH);
+        str = str.substr(0, MAX_PLACEABLE_LENGTH);
+      }
+
+      if (ctx.useIsolating) {
+        result += FSI + str + PDI;
```
Comment (zbraniecki) [inline src/intl/resolver.js]: template string here to avoid two concats?

Comment (stasm) [inline src/intl/resolver.js]: ok!

Comment (stasm) [inline src/intl/resolver.js]: @zbraniecki Just to clarify: do you mean:

    result += `${FSI}${str}${PDI}`;

or:

    result = `${result}${FSI}${str}${PDI}`;

?

Comment (zbraniecki) [inline src/intl/resolver.js]: I don't think latter has any impact, so former :)

**Pull request merged**